recipes/latexmk/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "latexmk" %}
+{% set major = "4" %}
+{% set minor = "55" %}
+{% set version = major + "." + minor %}
```

Comment:
Would it be possible to swap this (have the version number be the true number and split off the major and minor?

Reply:
I guess that `latexmk-{{{ version|replace(".", "") }}.zip` is cleaner, would that work?

Reply:
The major and minor are also used in the url

Reply:
I don't think that should matter. And the replace may not work in the future if they add a patch/micro number.

Reply:
I guess my main thing is that the bot will not be very friendly to this, although since the upstream source is not currently supported it doesn't matter too much.

Reply:
That seems like a deficiency in the autoticker-bot, rather than an issue with the recipe. I know for a fact that there are other recipes out there that split up version numbers like this. This seems like it is something that could be solved by having the recipe update customization

Reply:
Fair